src/openzaak/conf/includes/base.py

```diff
@@ -447,6 +447,9 @@
 
 # settings for uploading large files
 MIN_UPLOAD_SIZE = config("MIN_UPLOAD_SIZE", 4 * 2 ** 30)
+DOCUMENTEN_UPLOAD_CHUNK_SIZE = 4 * 2 ** 30  # 4 GB
```

Comment:
maybe this should be set to `MIN_UPLOAD_SIZE`, that's the envvar that also controls the maximum post body size at the nginx level.

If you set `MIN_UPLOAD_SIZE` at deployment to 2GB for example, but create bestandsdeel chunks for 4GB, clients will never be able to upload these files since nginx throws "Request Entity Too Large" errors.